Modules/Authentication/GenHTTP.Modules.Authentication.csproj

```diff
@@ -14,7 +14,7 @@
         <Version>9.3.0</Version>
 
         <Authors>Andreas Nägeli</Authors>
```

Comment:
Probably add yourself as an author there :)

Reply:
I'm good for now, thank you. I'm happy that I can give something back in return for such awesome project. ❤️